### PR TITLE
add kube-controller-manager target

### PR DESCRIPTION
**Pull request by TheoBrigitte**
Towards: https://github.com/giantswarm/giantswarm/issues/11724

image

image

workqueue_work_duration_seconds_sum{name="replicaset"} 1.5397501930000002

### CODE REVIEW

**Review by JosephSalisbury — COMMENTED** [review]
lgtmiiwfy

File: files/templates/additional-scrape-configs.template.yaml [hunk 1]
```diff
@@ -243,3 +243,36 @@
...
+  tls_config:
+    ca_file: /etc/prometheus/secrets/{{ .SecretName }}/ca
+    cert_file: /etc/prometheus/secrets/{{ .SecretName }}/crt
+    key_file: /etc/prometheus/secrets/{{ .SecretName }}/key
+    insecure_skip_verify: true
+  relabel_configs:
+  - source_labels: [__meta_kubernetes_pod_label_k8s_app]
+    regex: controller-manager
+    action: keep
+  - source_labels: [__meta_kubernetes_pod_name]
```
Comment (omegas27) [inline files/templates/additional-scrape-configs.template.yaml:271]: Why do we rename the pod name to app? Should the app name not be kubernetes-controller-manager or controller-manager instead?

Comment (TheoBrigitte) [reply files/templates/additional-scrape-configs.template.yaml:271]: something like this right?

```yaml
  - target_label: app
    replacement: kube-controller-manager
```

Comment (omegas27) [reply files/templates/additional-scrape-configs.template.yaml:271]: Yep exactly

Comment (TheoBrigitte) [reply files/templates/additional-scrape-configs.template.yaml]: done

**Review by omegas27 — APPROVED** [review]

**Pull request merged**